libexec/rtld-elf/aarch64/rtld_c18n_asm.S

```diff
@@ -47,11 +47,15 @@ ENTRY(_rtld_setjmp)
 	 * We store a sealed capability to the caller's frame in the jump
 	 * buffer.
 	 */
-	ldr	c2, sealer_jmpbuf
-	ldr	c3, [csp]
-	seal	c3, c3, c2
-	str	c3, [c0], #CAP_WIDTH
-	ret
+#ifdef __ARM_MORELLO_PURECAP_BENCHMARK_ABI
```

Comment:
Why does the !defined(__ARM_MORELLO_PURECAP_BENCHMARK_ABI) case look entirely different after this commit?